etc/whois.profile

```diff
@@ -45,8 +45,8 @@ private
 private-bin bash,sh,whois
 private-cache
 private-dev
-private-etc alternatives,hosts,jwhois.conf,services,whois.conf
-private-lib
+private-etc alternatives,hosts,jwhois.conf,services,whois.conf,resolv.conf
+private-lib gconv
```

Comment:
I'm trying to reproduce this. Plaese see #3153 for details.

Reply:
I get it on Debian stable:

$ firejail whois google.com
getaddrinfo(whois.verisign-grs.com): Name or service not known